packages/components/src/DataList/components/DataListLayoutInternal/DataListItems.tsx

```diff
@@ -24,104 +19,32 @@ export function DataListItems<T extends DataListObject>({
   mediaMatches,
   data,
 }: DataListItemsProps<T>) {
-  const { itemActionComponent } = useDataListContext();
-  const elementData = useMemo(() => generateListItemElements(data), [data]);
```

Comment:
ohhhhh so are you removing the memoization here because now it's being handled in `DataListItem`, which is being mapped over in this file now.  Therefore being handled by each item rather than the whole list of items...

Reply:
Nice! Great point!